lib/ui/compositing.dart

```diff
@@ -281,18 +281,19 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// See [pop] for details about the operation stack.
   TransformEngineLayer? pushTransform(
     Float64List matrix4, {
+    Float64List? cacheMatrix4,
```

Comment:
Perhaps name this just `cacheMatrix` and restrict it to 3x3 or 3x2 or 2x2 depending. The vec64 package only allows Matrix2,Matrix3,Matrix4 so 3x2 isn't really available...